snuba/clickhouse/http.py

```diff
@@ -291,6 +291,8 @@ def __init__(
         self,
         host: str,
         port: int,
+        max_connections: int,
```

Comment:
Please make these optional parameters with defaults, both for this class and the `ClickhouseCluster` class. We have settings files outside of this repo, and if we merge this all those clusters will break.

Reply:
Done